### DISCUSSION TITLE
How do I display a progress bar?

### DISCUSSION THREAD

**Discussion by icarosadero** [AdaptiveMotorControlLab/CEBRA#178] [Q&A]
I have been searching the docs for instructions on how to display the progress bar during training, but didn't find anything. It seems that it already has an implementation of it since it imported the tqdm package. Has anyone figured that out?

**Comment by MMathisLab** [comment]
It shows by default in Jupyter books; ie see demo notebooks :)

**Reply by icarosadero** [reply]
Hello,

It doesn't show up to me on VScode notebook. I am using this:

```
single_cebra_time = CEBRA(
    batch_size=512,
    output_dimension=8,
    device="cuda_if_available",
    distance="cosine",
    learning_rate=3e-4,
    time_offsets=10
)
U = single_cebra_time.fit_transform(X)
```

**Comment by icarosadero** [comment]
I noticed I just have to pass `verbose=True`, which is disabled by default.
